package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "hexal-task",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack serve"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LonelIce/hexal-task.git"
+  },
+  "keywords": [
+    "Sass",
+    "layout",
+    "HTML",
+    "Webpack",
+    "Git",
+    "CSS"
+  ],
+  "author": "Ivan Moroz <ijumrz@gmail.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/LonelIce/hexal-task/issues"
+  },
+  "homepage": "https://github.com/LonelIce/hexal-task#readme",
+  "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/preset-env": "^7.15.0",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "sass": "^1.38.1",
+    "sass-loader": "^12.1.0",
+    "style-loader": "^3.2.1",
+    "webpack": "^5.51.1",
+    "webpack-cli": "^4.8.0",
+    "webpack-dev-server": "^4.0.0"
```

Comment:
Check the difference between "devDependencies" and "dependencies". And install the dependencies into the correct section.